services/provider/server/server.go

```diff
@@ -366,21 +368,6 @@ func (s *OCSProviderServer) getExternalResources(ctx context.Context, consumerRe
 				return nil, fmt.Errorf("failed to get %s secret. %v", clientSecretName, err)
 			}
 
-			idProp := "userID"
-			keyProp := "userKey"
-			if cephClaim == "cephfs" {
-				idProp = "adminID"
-				keyProp = "adminKey"
-			}
-			extR = append(extR, &pb.ExternalResource{
-				Name: clientSecretName,
-				Kind: "Secret",
-				Data: mustMarshal(map[string]string{
-					idProp:  cephRes.Name,
-					keyProp: string(cephUserSecret.Data[cephRes.Name]),
-				}),
-			})
```

Comment:
Without this code you will never create the health checker client secret

Reply:
good catch, missed this one added it now.